### ISSUE TITLE
Consistent color systems across the library

### ISSUE THREAD

**Issue by KrishnaMoorthy12** [NehemiahK/fun-react-lib#16]
Let's define colors for the entire library to maintain same color palatte, otherwise the components will become inconsistent and look very different from each other. We can also move to sass for a hassle free implementation. I request everyone reading this to leave a comment on this..

**Comment by ferueda** [comment]
Here are a few color palette pickers that might help with the process.

https://flatuicolors.com/

https://coolors.co/palettes/trending

Personallt, I like the following palette theme

```CSS
#e63946;
#f1faee;
#a8dadc;
#457b9d;
#1d3557;
```

**Comment by NehemiahK** [comment]
Yes, I think this is a great idea. Should probably go into the App.css file.

Anyone want to get this started?

**Comment by ferueda** [comment]
> Yes, I think this is a great idea. Should probably go into the App.css file.
> 
> Anyone want to get this started?

I think we should make like a voting process before implementing it?

**Comment by jessecookedesign** [comment]
I'm happy to help on this.

I'm impartial to the colors used but I'd suggest using CSS variables.

Example
```html
:root {
 --color-red: #e63946;
}
```

**Comment by KrishnaMoorthy12** [comment]
I suggest to go with sass, for a better syntax and easy importing... Also, it'll be easy for us to write utility classes.

**Comment by ashispavan** [comment]
> I suggest to go with sass, for a better syntax and easy importing... Also, it'll be easy for us to write utility classes.

Agreed.

As for color palettes, we could pick a scheme from here: https://colorsinspo.com/

**Comment by KrishnaMoorthy12** [comment]
> > I suggest to go with sass, for a better syntax and easy importing... Also, it'll be easy for us to write utility classes.
> 
> Agreed.
> 
> As for color palettes, we could pick a scheme from here: https://colorsinspo.com/

That looks great. We'll choose one.. @NehemiahK , your opinion?

**Comment by KrishnaMoorthy12** [comment]
We need to do this first. Then only we can start working on other components. We need to make a quick decision.

**Comment by NehemiahK** [comment]
Sorry, wasn't at my computer. I'm open to whichever scheme of the person that want's to tackle this

**Comment by ashispavan** [comment]
@KrishnaMoorthy12  You can pick a scheme of your liking. Ideally, we could have a ThemeProvider created so we could easily swap color palettes by changing a single prop, but for now you can just pick one for the sake of proceeding with development of other components like you mentioned.

**Comment by KrishnaMoorthy12** [comment]
Okay, I'll try to add the theme provider component tomorrow.
